src/js/claims-status/routes.jsx

```diff
@@ -41,10 +42,19 @@ const routes = [
     component={AppealLayout}
     key="/appeals-v2"
     path="/appeals-v2">
+    {/* New Routing Start */}
+    <Redirect
+      key=":id"
+      from=":id"
+      to=":id/status"/>
     <Route
-      component={AppealsV2StatusPage}
+      component={AppealInfo}
       key=":id/status"
       path=":id/status"/>
+    <Route
+      component={AppealInfo}
```

Comment:
Can we wrap these `Route`s in another `Route` that uses `AppealInfo` as its component and have that just renders its children?

```jsx
<Route
  component={AppealInfo}
  key=":id"
  path=":id">
  <Redirect
    key="/"
    from="/" {/* May need to be an IndexRedirect or something; I don't remember */}
    to="/status"/>
  <Route
    component={AppealsV2StatusPage}
    key="/status"
    path="/status"/>
  <Route
    component={AppealsV2DetailsPage}
    key="/details"
    path="/details"/>
</Route>
```

This way, we won't have to do any route-based rendering from inside `AppealInfo`, y'know?

Reply:
🤔 Let me look into that. I'm all for reducing complexity in the components.